src/models/ModulePart.ts

```diff
@@ -0,0 +1,15 @@
+import { ModulePartElement } from './ModulePartElement';
+
+export class ModulePart {
```

Comment:
can use TS type instead of class if there is not instanciation needed 
ex :

```
type ModulePart = { 
    image: string;
    order: number;
    subtitle: string;
    elements: Array<ModulePartElement>;
};
```